src/ngraph/runtime/tensor.hpp

```diff
@@ -20,6 +20,7 @@
 #include <vector>
 
 #include "ngraph/descriptor/tensor.hpp"
+#include "ngraph/descriptor/layout/tensor_layout.hpp"
```

Comment:
is this needed ?

Reply:
Yes. Without this there is a compile error.